scripts/generate_code.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+CDIR=$(dirname $0)
+XDIR=$(realpath $CDIR/..)
+PTDIR=$(realpath $XDIR/..)
+
+$CDIR/gen.py \
```

Comment:
We have to run it in the Python where we install lark.

Reply:
Done.